src/projections/s2.cpp

```diff
@@ -0,0 +1,341 @@
+/*
```

Comment:
Please include a header with license & copyright info (you can take src/transformations/defmodel.cpp as example, and adapt)